client/debug/main.go

```diff
@@ -98,22 +100,25 @@ func PubkeyCmd(cdc *codec.Codec) *cobra.Command {
 			if err != nil {
 				return err
 			}
-			fmt.Println("Address:", pubKey.Address())
-			fmt.Printf("Hex: %X\n", pubkeyBytes)
-			fmt.Println("JSON (base64):", string(pubKeyJSONBytes))
-			fmt.Println("Bech32 Acc:", accPub)
-			fmt.Println("Bech32 Validator Operator:", valPub)
-			fmt.Println("Bech32 Validator Consensus:", consenusPub)
+			cmd.Println("Address:", pubKey.Address())
+			cmd.Printf("Hex: %X\n", pubkeyBytes)
+			cmd.Println("JSON (base64):", string(pubKeyJSONBytes))
+			cmd.Println("Bech32 Acc:", accPub)
+			cmd.Println("Bech32 Validator Operator:", valPub)
+			cmd.Println("Bech32 Validator Consensus:", consenusPub)
 			return nil
 		},
 	}
 }
 
 func AddrCmd() *cobra.Command {
 	return &cobra.Command{
-		Use:   "addr",
+		Use:   "addr [address]",
 		Short: "Convert an address between hex and bech32",
-		Args:  cobra.ExactArgs(1),
+		Long: `Convert addresses:
+		
```

Comment:
This won't be printed to screen as expected, you should deindent this line. I suggest to take inspiration from other commands, e.g. https://github.com/cosmos/cosmos-sdk/blob/master/x/distribution/client/cli/tx.go#L91